Tests/LibWeb/Text/expected/css/getComputedStyle-print-all.txt

```diff
@@ -120,7 +121,7 @@ grid-row-start: auto
 grid-template-areas: 
 grid-template-columns: 
 grid-template-rows: 
-height: 2074px
```

Comment:
introduction of writing-mode support should not affect used height. where is this change coming from?

Reply:
As I understand this comes from rebaselining this test. Sam mentioned this in the discord [here](https://discord.com/channels/1247070541085671459/1247090082297548832/1291023875400007720)

Reply:
got it, seems correct!